.github/workflows/ci.yml

```diff
@@ -90,7 +90,7 @@ jobs:
     - name: Setup PHP
       uses: shivammathur/setup-php@v2
       with:
-        php-version: '8'
+        php-version: '8.1'
```

Comment:
I went through the Changelog for 8.1, there is now native Enumeration in PHP.
[This discussion](https://github.com/spatie/enum/issues/90) might also be relevant.
I will look into this, but if you already thought about this and strongly want to keep the spatie package, feel free to intervene. In any case, only tangential related to this PR...

Reply:
I’m definitely in favor of using native enums over a library, but so far, I’ve been too lazy to look into what’s necessary to migrate… :)